programs/voter-stake-registry/tests/program_test/mod.rs

```diff
@@ -110,7 +110,8 @@ impl TestContext {
         let mut test = ProgramTest::new(
             "voter_stake_registry",
             addin_program_id,
-            processor!(voter_stake_registry::entry),
+            // processor!(voter_stake_registry::entry),
+            None,
```

Comment:
why would you not bind the entry point?

Reply:
new anchor complains due to mismatched types. 
![image](https://github.com/user-attachments/assets/50908588-6839-494c-8ef3-66ade75dd8dc)

solana-program inherits the entry from the .so if the .so files exists. 
I couldn't find another way to adding the builtin_function.